modules/javafx.graphics/src/main/java/javafx/scene/layout/AnchorPane.java

```diff
@@ -280,8 +280,7 @@ private double computeWidth(final boolean minimum, final double height) {
                     child.minWidth(childHeight) : computeChildPrefAreaWidth(child, -1, null, childHeight, false)) + right);
         }
 
-        final Insets insets = getInsets();
-        return insets.getLeft() + max + insets.getRight();
+        return snappedLeftInset() + max + snappedRightInset();
```

Comment:
Have you verified that `max` ends up as a snapped value in all cases? Some of the values that are used in the computation of max clearly are snapped, such as `computeChildPrefAreaWidth` (which is a method in the Region superclass that returns a snapped result), but others might not be.

Reply:
besides the anchor everything is snapped properly here already.